src/EnergyPlus/api/runtime.cc

```diff
@@ -73,6 +73,10 @@ int energyplus(int argc, const char *argv[]) {
     return runEnergyPlusAsLibrary(state, argc, argv);
 }
 
+void stopSimulation() {
+  EnergyPlus::DataGlobals::stopSimulation = true;
```

Comment:
It would be nice to make this stopSimulation a function call so that clients don't have to mine their way into the DataGlobals variable.

Reply:
https://github.com/NREL/EnergyPlus/blob/APIs-and-spawn2/src/EnergyPlus/api/runtime.h#L121 it looks like the function needs some documentation though. I'll add something.

Reply:
Something other than the stopSimulation() function? I figured they would use that. The DataGlobals::stopSimulation was not intended to be used publicly.

Reply:
Yeah I dunno what I missed when I commented this.